### PR TITLE
Upgrade to ganache-cli@istanbul for testing with Istanbul

### PR DESCRIPTION
**Pull request by Anyhowclick**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "ethereumjs-abi": "^0.6.8",
     "ethereumjs-tx": "^2.1.0",
     "ethereumjs-util": "^6.1.0",
-    "ganache-cli": "^6.7.0",
+    "ganache-cli": "^6.8.0-istanbul.0",
     "mathjs": "^4.3.0",
```
Comment (ilanDoron) [inline package.json]: this version default version is without Istanbul?

File: README.md [hunk 2]
```diff
@@ -13,16 +13,11 @@ For more details, please visit our [developer portal](https://developer.kyber.ne
...
 ### Example Commands
 `./tst.sh -f "./test/kyberReserve.js"`
-
-## Testing with Istanbul Hardfork
-As of the time of writing, `ganache-cli`  does not yet support the Istanbul hardfork. Kindly perform the following steps to run ganache manually.
-1. Clone the [ganache-cli repo](https://github.com/trufflesuite/ganache-cli/)
-2. CD into the repo and `npm i`
-3. `node cli.js -k istanbul`
-4. In another terminal, cd into this repo. To test against the full suite, simply do `npx buidler test --no-compile`. Otherwise, do `npx buidler test --no-compile {FILE_DIRECTORY}`
+`./tst.sh -f "./test/kyberReserve.js" -k istanbul`
```
Comment (ilanDoron) [inline README.md]: I am using ./tst.sh -k Istanbul
I am not sure it really runs Istanbul.
please print some feed back to show it is in did running istanbul

Comment (ilanDoron) [reply README.md]: I tried with -k istanbul
and I get no failure with kyberNetwrokProxy.js
so not sure if its running.

File: package.json (hunk 1 above)
Comment (Anyhowclick) [reply package.json]: Yup. Runs on Petersburg by default.

File: README.md (hunk 2 above)
Comment (Anyhowclick) [reply README.md]: Wrong logic, needed to switch the statements around. Fork choice is printed if you're running on non-default fork (default is Petersburg)

Comment (Anyhowclick) [reply README.md]: Tests should break because Istanbul fix not included.

**Pull request merged**